features/context.feature

```diff
@@ -310,7 +310,7 @@ Feature: Context consistency
     Then it should fail with:
       """
       [Behat\Testwork\Suite\Exception\SuiteConfigurationException]
-        `contexts` setting of the "default" suite is expected to be an array, string given.
+        `contexts` setting of the "default" suite is expected to be an array, `string` given.
```

Comment:
`gettype` could return `unknown type` or `NULL` but apparently it will [never return an empty string](http://php.net/manual/en/function.gettype.php). So I'm fine with reverting that.